lib/reporter.ts

```diff
@@ -201,7 +201,8 @@ export class CucumberJsJsonReporter extends WDIOReporter {
      */
     public onRunnerEnd (): void {
         const jsonFolder = resolve( process.cwd(), this.options.jsonFolder );
-        const jsonFile = resolve( jsonFolder, `${this.report.feature.id}.json` );
+        const date = new Date(Date.now()).toISOString().replace(/:/g,"-").split(".")[0];
+        const jsonFile = (0, path_1.resolve)(jsonFolder, `${this.report.feature.id}_${date}.json`);
```

Comment:
Hi @osmolyar 
This will not work because it's a compiled code in a uncompiled TypeScript file.

I think you wanted to do something like this

```
const date = new Date(Date.now()).toISOString().replace(/:/g,"-").split(".")[0];
const jsonFile = resolve( jsonFolder, `${this.report.feature.id}_${date}.json` );
```


you also want to post-fix the name like `${this.report.feature.id}_2022-01-31T16-41-00.json`

Reply:
Hi @wswebcreation , you're right, I'd put that code in the reporter.js file locally to test it, and thought it might compile from the ts.  How should this be handled?

Re. 
> you also want to post-fix the name like ${this.report.feature.id}_2022-01-31T16-41-00.json

Isn't that what 
`const jsonFile = (0, path_1.resolve)(jsonFolder, `${this.report.feature.id}_${date}.json`);`

is doing (if present in compiled file).

Reply:
Ohh, too early for reading comprehension 😆   Thanks!